.github/workflows/xfstests.yaml

```diff
@@ -0,0 +1,101 @@
+name: xfstests
```

Comment:
please drop this commit as it will trigger for any other repo commit and if not needed ci.yaml commit as well

Reply:
sounds good